bukkit/src/main/java/net/william278/husktowns/hook/PlaceholderAPIHook.java

```diff
@@ -349,6 +286,45 @@ private Optional<String> resolveTownMemberName(@NotNull Town town, @NotNull UUID
                     .findFirst();
         }
 
+        @Nullable
+        public Object resolveTownData(@NotNull Town town, @NotNull String identifier) {
+            return switch (identifier) {
```

Comment:
Could be wrong but I don't think you're using this method anywhere where you're not immediately converting the object to a String - so perhaps have this return a Nullable or `Optional<>` String instead?